### PR TITLE
411: Ping API documentation

### PR DESCRIPTION
**Pull request by schippada**

### CODE REVIEW

File: blueprint/api.apib [hunk 1]
```diff
@@ -133,6 +133,26 @@ Exchange credentials for `access_token`.
 
 # Group Ping
 
+You can check if the server is up and running by hitting the `/ping` end-point. The response differs between logged in user and the anonymous user as shown below
```
Comment (joshsmith) [inline blueprint/api.apib:136]: Prefer wording of `endpoint` and trailing period.

File: blueprint/api.apib [hunk 2]
```diff
@@ -133,6 +133,26 @@ Exchange credentials for `access_token`.
...
+
+## Ping [/ping]
+
+### Check server status [GET]
+
+For a logged in users, this end-point returns the users registered email id in the response json.
```
Comment (joshsmith) [inline blueprint/api.apib:142]: "user", "endpoint", and if you can write it as:

```
`id`
```

and "JSON".

Also prefer "authenticated" over "logged in" for consistency.

File: blueprint/api.apib [hunk 3]
```diff
@@ -133,6 +133,26 @@ Exchange credentials for `access_token`.
 
 # Group Ping
 
+You can check if the server is up and running by hitting the `/ping` end-point. The response differs between logged in user and the anonymous user as shown below
+
+## Ping [/ping]
+
+### Check server status [GET]
```
Comment (joshsmith) [inline blueprint/api.apib:140]: > Check authenticated...

File: blueprint/api.apib [hunk 4]
```diff
@@ -133,6 +133,26 @@ Exchange credentials for `access_token`.
...
+
+For a logged in users, this end-point returns the users registered email id in the response json.
+
++ Response 200 (application/json)
+
+  + Attributes (Ping Logged In User Response)
+
+### Check server status [GET]
```
Comment (joshsmith) [inline blueprint/api.apib:148]: > Check authenticated...

File: blueprint/api.apib [hunk 5]
```diff
@@ -133,6 +133,26 @@ Exchange credentials for `access_token`.
...
+
+For a anonymous users, this end-point returns `pong` in the response json.
```
Comment (joshsmith) [inline blueprint/api.apib:150]: remove leading "a", "endpoint", "JSON"

File: blueprint/api.apib [hunk 6]
```diff
@@ -330,3 +350,12 @@ Exchange credentials for `access_token`.
...
   + data(array[User Skills Relationship], required)
+
+
+## Ping Guest User Response (object)
++ data
+  + ping: `pong` (string)
+
+## Ping Logged In User Response (object)
+ + data
+   + ping: `author@coderly.com` (string)
```
Comment (joshsmith) [inline blueprint/api.apib:361]: Super trivial, but you have a couple leading ` `spaces before`data`and`ping`. Also would suggest`team@codecorps.org`, which is an actual email people can reach us at. :D

**Comment by joshsmith** [comment]
With those changes made, you can squash your commits and merge without review. Although if you'd like another review as a sanity check, happy to give one.

**Comment by npendery** [comment]
@schippada, not sure if the 411 in your title will close the issue, but if not would you mind adding a `Closes #411` in your PR description and maybe just a small list of changes made? Thanks!

**Comment by joshsmith** [comment]
@schippada can you implement @npendery and my comments above when you have a moment?

**Comment by schippada** [comment]
@JoshSmith & @npendery , I didn't know that I need to look here. That is the reason why I didn't make these fixes. Apologies for that. I create a totally new pull request. I will close this pull request to avoid any confusion.

**Pull request closed without merging**